packages/abi-gen/src/index.ts

```diff
@@ -120,6 +107,14 @@ for (const abiFileName of abiFileNames) {
         process.exit(1);
     }
 
+    const outFileName = utils.makeOutputFileName(namedContent.name);
+    const outFilePath = `${args.output}/${outFileName}.ts`;
+
+    if (utils.isOutputFileUpToDate(abiFileName, outFilePath)) {
+        logUtils.log('Already up to date: ' + outFilePath);
```

Comment:
Nit: Please use `template strings` over concatenation everywhere. This would be:

```
logUtils.log(`Already up to date: ${outFilePath}`);
```

Reply:
Once this get's fixed, we will merge this :)

Reply:
All fixed!  But before merging do check with @LogvinovLeon on his opinion on the other change I made at the same time (explained in comments below).